app/css/general.css

```diff
@@ -83,16 +84,18 @@ html,body {
 	border-right: 4px solid #F5F5F5;
 }
 .sidebar .button {
+	opacity: .7;
 	display:flex;
 	font-size: 16px;
 	align-items:center;
 	position: relative;
 	height: 60px;
 
+	transition: all .25s;
+	-webkit-animation: fadein 2s;
```

Comment:
I don't know if I'm making this number up, but I thought 400ms was a standard upper bound on stuff like this.